test/clubbing_top_level.txt

```diff
@@ -0,0 +1 @@
+<table border="1"><tr><th>aaa</th><th>ccc</th></tr><tr><td>1</td><td>2</td></tr><tr><td>bbb</td><td>2</td></tr></table>
```

Comment:
Sure, I'll pick it up with few other modifications soon.

Just for the sake of consistency, I think it should return with ul>li because a user expects the same in return i.e. a list. I'm not sure why you want it the other way.

If it looks good to you, then you can skip it for the time being.

Rest assured, PR is good to be merged then.